referrer-policy/generic/multiple-headers-one-invalid.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Referrer Policy: multiple Referrer-Policy headers with one invalid</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+
+    <script src="/referrer-policy/generic/common.js"></script>
+  </head>
+  <body>
+    <h1>Referrer Policy: multiple Referrer-Policy headers with one invalid</h1>
+    <p></p>
+
+    <pre id="received_message">Running...</pre>
+
+    <script>
+      var test = async_test("Image uses the last recognized valid Referrer-Policy header");
+      var urlPath = '/referrer-policy/generic/subresource/image.py?cache_destroyer=' + (new Date()).getTime();
+      var url = location.protocol + "//www1." + location.hostname + ":" + location.port +
+                urlPath;
+      queryImage(url, test.step_func(function(message) {
+        assert_equals(message.referrer, document.location.origin + "/");
```

Comment:
(Also, I have to update the Chrome implementation to match this behavior: https://crbug.com/705950)